### DISCUSSION TITLE
Mp3 thumbnail

### DISCUSSION THREAD

**Discussion by ghost** [JunkFood02/Seal#250] [Q&A] [answered]
Is there any way I can use thumbnail for each mp3 as their thumbnail. 
My music player always shows one thumbnail for album art and all mp3. 
So can i get through this ?

**Comment by Marco-9456** [comment, marked as answer]
> Is there any way I can use thumbnail for each mp3 as their thumbnail. My music player always shows one thumbnail for album art and all mp3. So can i get through this ?

I think it's depending on the Music player you are using.

**Reply by Marco-9456** [reply]
@IamHARSHDABAS here you go https://play.google.com/store/apps/details?id=com.piyush.music

**Reply by ghost** [reply]
It worked thx

**Reply by Marco-9456** [reply]
> It worked thx

You're welcome

**Comment by Donkey-Doug** [comment]
Hi, I was not sure if I should make a seperate issue, so I decided to mention it here:

Downloading audio takes much longer than downloading a video. 

I don't know if this is a bug, but the difference in time is massive. I would expect music to download faster.

**Reply by JunkFood02** [reply]
Please start a new discussion
